streams/src/test/java/org/apache/kafka/streams/processor/internals/metrics/ThreadMetricsTest.java

```diff
@@ -413,6 +415,39 @@ public void shouldAddThreadStartTimeMetric() {
         );
     }
 
+    @Test
+    public void shouldAddThreadStateTelemetryMetric() {
+        final Gauge<Integer> threadStateProvider = (streamsMetrics, startTime) -> StreamThread.State.RUNNING.ordinal();
+        ThreadMetrics.addThreadStateTelemetryMetric(
+                "threadId",
```

Comment:
There is a constant for a test thread ID:

```suggestion
                THREAD_ID,
```